### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Ayokunnumi1**
These are the functionalities implemented in this branch;
- Homepage
- Comment PopUp
- Use of two API
- Updated Read Me file
- Video links
- Testing

### CODE REVIEW

File: src/modules/display.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  });
+};
+
+// Function to update likes
+const updateLikesCount = (id) => {
+  const likesCount = document.querySelector(`.likes-count[id="${id}"] > span`);
+  const likesCountValue = parseInt(likesCount.textContent, 10);
+  likesCount.textContent = `${likesCountValue + 1}`;
+};
+
```
Comment (divinecharlotte) [inline src/modules/display.js]: - [ ] kindly consider separating the counters in their own modules to meet the below requirement 👇 
> Each counter should be implemented as a separate module.

File: src/modules/display.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+}
+const initializeApp = async () => {
+  const movies = await fetchShows(url);
+  const likes = await getLikes(likesUrl);
+  const filteredShowObject = processShowData(movies, likes);
+  const movieLength = calculateTotalMovies(filteredShowObject);
+  const totalMovies = document.querySelector('.total-movies');
+  totalMovies.textContent = ` All movies (${movieLength})`;
+  renderMovies(filteredShowObject);
+  addEventToLikeButtons();
```
Comment (divinecharlotte) [inline src/modules/display.js:85]: - [ ] kindly consider separating the counters in their own modules to meet the below requirement 👇 
> Each counter should be implemented as a separate module.

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team    , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Clean code👍 
- Implemented some project requirements   👍 

 




 

## Required Changes ♻️

- [ ] Kindly consider adding a working link to the social media of the second partner in your readme to make it more professional and that will contribute to giving a good user experience to the people visiting your readme mainly employers or people who might get interested in your project and want to reach you out
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/display.js (hunk 1 above)
Comment (Whoistolu) [reply src/modules/display.js]: - Well resolved

File: src/modules/display.js (hunk 2 above)
Comment (Whoistolu) [reply src/modules/display.js:85]: - Well resolved

File: README.md [hunk 3]
```diff
@@ -1,2 +1,156 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/https://github.com/Ayokunnumi1).
```
Comment (Whoistolu) [inline README.md]: - [ ] [OPTIONAL] Kindly change the issues link as this attached link is currently broken. The issue link would be used by any developer that discovers and wants to raise an issue concerning the project.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,156 @@
...
+
+- I would like to give thanks to Microverse for creating this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/https://github.com/Ayokunnumi1/dev/LICENSE) licensed.
```
Comment (Whoistolu) [inline README.md]: - [ ] [OPTIONAL] Kindly replace the MIT License link with a working link as this particular link is currently broken. The License is important to declare to other developers that you own this project and it is a hard requirement.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,156 @@
...
+
+- **TV Shows Listing: Browse a collection of popular TV shows with their details, such as title, description, and airing date.**
+
+- **Like Shows: Users can like their favorite TV shows, and the app will keep track of their preferences.**
+
+- **Comments: Users can leave comments on TV shows to share their opinions and discuss with other users.**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Visit our live project]()
```
Comment (Whoistolu) [inline README.md:55]: - [ ] [OPTIONAL] Kindly remove this section of README since it is not required and this project has not deployed.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,156 @@
...
+
+## 🚀 Description Video<a name="video-demo"></a>
+
+- [Link to our description video](https://drive.google.com/file/d/17jh4JKd81T7QvOJDLy_CoZUjgQS_UlN_/view?usp=sharing)
+
```
Comment (Whoistolu) [inline README.md]: - [ ] [OPTIONAL] Great job concerning the video presentation even though it is no longer a requirement for this capstone. However, always ensure a consistently professional tone and avoid any laughter or disruptions that could detract from the project's credibility. Also, you can carefully review and edit the video to eliminate any unintended moments of laughter or informality. Consider rehearsing to enhance overall presentation quality.

File: src/test/commentCount.test.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import { calculateCommentLength } from '../modules/popUp.js';
```
Comment (Whoistolu) [inline src/test/commentCount.test.js:1]: - [ ] [OPTIONAL] Part of the project requirement states that you should have tests for all edge cases. You could think at least about the scenario in which there are 0 items on the page (counter function should return 0)

**Review by Whoistolu — APPROVED** [review]
### Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Ayokunnumi1) [reply README.md]: Ok thank you, I have worked on it.

File: README.md (hunk 4 above)
Comment (Ayokunnumi1) [reply README.md]: Alright, i have worked on this as well, thank you

File: README.md (hunk 6 above)
Comment (Ayokunnumi1) [reply README.md]: I have edited out the unnecessary parts of the video as well

File: src/test/commentCount.test.js (hunk 7 above)
Comment (Ayokunnumi1) [reply src/test/commentCount.test.js:1]: I have worked on this as well, thank you
